tests/src/test_helpers/ssl_helpers.c

```diff
@@ -806,13 +806,14 @@ int mbedtls_ssl_write_fragment(mbedtls_ssl_context *ssl,
                                int *written,
                                const int expected_fragments)
 {
+    int ret;
```

Comment:
Please separate variable declarations from function body by a blank line (there's are two other instances of this)